packages/e2e/test/k6/endpoints/asset/get-assets.test.js

```diff
@@ -1,278 +1,155 @@
+import { SharedArray } from 'k6/data';
+import { apiVersion } from '../../../../../cardano-services-client/src/version.ts';
+import { check, fail } from 'k6';
+import exec from 'k6/execution';
 import http from 'k6/http';
 
-// eslint-disable-next-line no-undef
-const PROVIDER_SERVER_URL = __ENV.PROVIDER_SERVER_URL;
+/**
+ * # Script description:
+ * - Sends REQUEST_COUNT HTTP requests to fetch assets. Since the number of iterations is indirectly controlled by the
+ *   duration and rate, it is possible to be slightly inaccurate.
+ * - Requests are started 1/s. A maximum of 20 VUs are used, so the maximum number of in-flight requests is 20.
+ * - The total number of assets requested per request is the sum of the ASSETS_* variables
+ * - AssetIds used are taken from mainnet. If the requested number of assets for the whole test is estimated to be larger
+ *   than the sample, the test will fail.
+ * - Test verifies that:
+ *   - Threshold for FAIL responses not exceeded (1% for example)
+ *   - Threshold for response having the wrong number of assets compared to requested is not exceeded (1% for example)
+ *   - 95th percentile of response time is less than 500ms
+ */
+
+const ASSETS_NO_METADATA_PER_REQUEST = Number.parseInt(__ENV.ASSETS_NO_METADATA_PER_REQUEST || 1);
+const ASSETS_ON_CHAIN_METADATA_PER_REQUEST = Number.parseInt(__ENV.ASSETS_ON_CHAIN_METADATA_PER_REQUEST || 0);
+const ASSETS_OFF_CHAIN_METADATA_PER_REQUEST = Number.parseInt(__ENV.ASSETS_OFF_CHAIN_METADATA_PER_REQUEST || 0);
+const REQUESTS_COUNT = Number.parseInt(__ENV.REQUESTS_COUNT || 1);
+const TARGET_ENV = __ENV.TARGET_ENV || 'dev'; // only dev and live are supported
```

Comment:
It would be nice to prevent/fail trying to run unsupported envs and networks.

Reply:
I've added the validation in 9e97339ec606635f256b581195b6a2fe45f08782 @ginnun